src/xml_comment.cc

```diff
@@ -109,7 +109,7 @@ void
 XmlComment::Initialize(v8::Handle<v8::Object> target)
 {
     NanScope();
-    v8::Local<v8::FunctionTemplate> t = NanNew<v8::FunctionTemplate>(New);
+    v8::Local<v8::FunctionTemplate> t = NanNew<v8::FunctionTemplate>(static_cast<NAN_METHOD((*))>(New));
```

Comment:
There are two overloaded New methods and it gave an error about being unable to pick which one. There is probably a new overload of NanNew<FunctionTemplate> that is causing the problem. Alternatively one of the methods could be renamed.